### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

**Comment by monstermunchkin** [comment]
@stgraber can we remove Ubuntu Xenial? It's not EOL (but neither was Trusty) but standard support has ended.

**Comment by stgraber** [comment]
@monstermunchkin I'd keep it another 6 months at least as we still need it ourselves for testing, especially to test upgrading away from it

File: jenkins/jobs/image-apertis.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@
         type: user-defined
         values:
         - 'v2019.7'
-        - 'v2020.4'
-        - 'v2021.0'
+        - 'v2020.5'
+        - 'v2021.1'
```
Comment (stgraber) [inline jenkins/jobs/image-apertis.yaml:23]: Can we detect this somehow and have it be handled like the RHEL derivatives?
So instead having images for `v2020` and `v2021` and have those pick whatever the latest point release is?

Right now every time we bump, it means users need to change their scripts to pull the new version which isn't ideal.

Comment (monstermunchkin) [reply jenkins/jobs/image-apertis.yaml:23]: Sure, I'll have distrobuilder find the latest release.

Comment (monstermunchkin) [reply jenkins/jobs/image-apertis.yaml:23]: Actually, distrobuilder already supports this.

**Pull request merged**